docs/faqs.md

```diff
@@ -0,0 +1,34 @@
+# Frequently Asked Questions
+
+## The generated JWT token can be decode, is this correct and secure?
```

Comment:
decode is missing a d at the end => "can be decoded". I'd also refrase the paragraph a bit. First of all the hash part should probably be encryption since a hash can not be reversed, its a oneway operation. Might also be a good idea to fuse this and the next paragraph. I'd write something along the lines of:
```
This is the expected behaviour. While the integrity of tokens is ensured by the generated/verified hash,
the contents of the token are only encoded and not encrypted. This means you can be sure the token
has not been modified by an unauthorized party, but you can not store confidential information in it. 
Anyone with access to the token can read all the claims you put into it. They can however not modify
them unless they have the (private or symetric) key used to generate the token. If you need to put
confidential information into it, you have to encrypt it beforehand using some other method, though
even then I recommend against doing it. Instead generate a random id and store the data on your
server, using the id to look it up whenever you need.
```

Reply:
```suggestion
## The generated JWT token can be decoded, is this correct and secure?
```